### ISSUE TITLE
Cannot read property 'firstChild' of null

### ISSUE THREAD

**Issue by ael-computas** [jdbranham/grafana-diagram#136]
This error message appears when I do something that produces an error in the plugin.  

For instance, if i misclick on Diagram definition to url, this error message appears and does not go away until i refresh the page.

**Comment by forestjohnsonpeoplenet** [comment]
It looks like this happens when the mermaid  markup code has a syntax error. I see this in my browser console when this happens on my graph. 

```
Panel data error: Error: Parse error on line 7:
... "a"subgraph my pod-thing
---------------------^
Expecting 'SEMI', 'NEWLINE', 'EOF', 'DIR', 'DOWN', 'SQS', 'MINUS', 'NUM', 'COMMA', 'ALPHA', 'COLON', 'BRKT', 'DOT', 'PUNCTUATION', 'UNICODE_TEXT', 'PLUS', 'EQUALS', 'MULT', got 'SPACE'
    at Parser.parseError (flow.js:329)
    at Parser.parse (flow.js:399)
    at Object.getClasses (flowRenderer.js:223)
    at Object.render (mermaidAPI.js:393)
    at DiagramCtrl.renderDiagram (diagramControl.js:347)
    at DiagramCtrl.updateDiagram (diagramControl.js:370)
    at DiagramCtrl.onDataReceived (diagramControl.js:171)
    at u.emit (index.js:129)
    at e.emit (emitter.ts:11)
    at DiagramCtrl.t.handleQueryResult (sdk.ts:1)
```

**Comment by sastorsl** [comment]
I found this on different markup errors.
I.e. mermaid supports `([text])` but Diagram does not and gives the above error.
Also spaces in subgraphs.

**Comment by ps-mariotacke** [comment]
Same thing here. What's really disruptive is that once it is in this "error" state, you have to refresh which may cause you to lose your configuration.

**Comment by ganadara135** [comment]
I solve this problem temporarily as reloading current page

**Comment by jdbranham** [comment]
I've added some error handling that should keep from 'tainting' the session.
It should be resolved in v1.6.2-rc1

https://github.com/jdbranham/grafana-diagram/releases

**Comment by jdbranham** [comment]
This is resolved in latest published version
